### PR TITLE
Changes/fixes to open existing map

### PR DESCRIPTION
**Pull request by macca400**
-Changes to open existing map sample to remove search and portal functionality. 
-Kept hard coding to a minimum. 
-Changed sample class name to follow naming convention followed by other .Net samples
-Created new branch rather than updating old branch to avoid Desktop/desktop folder fiasco

### CODE REVIEW

**Comment by macca400** [comment]
@anttikajanus I've made the CR changes you mentioned. I've opened this new PR for final merge to avoid the Desktop/desktop windows folder issue that would have been caused by the old branch. 

Have a look and then I'll merge

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMapVB.xaml.vb [hunk 1]
```diff
@@ -0,0 +1,100 @@
+﻿
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMapVB.xaml.vb]: make sure that we don't have empty lines at the top of the file

File: src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMapVB.xaml.vb [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            InitializeComponent()
+            AddHandler Loaded, AddressOf OnLoaded
+
+        End Sub
+
+        ' Loads UI elements and an initial Map.
+        Private Async Sub OnLoaded(sender As Object, e As RoutedEventArgs)
+
+            ' Adding items' Titles and URLs to a collection that will be used to populate the combobox's drop down. 
+            Dim comboBoxContent As ICollection(Of KeyValuePair(Of [String], [String])) = New Dictionary(Of [String], [String])() From {
```
Comment (anttikajanus) [inline src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/OpenExistingMap/OpenExistingMapVB.xaml.vb]: Could you make the row a bit shorter

**Comment by anttikajanus** [comment]
Looks very nice. I added couple trivial comments. Please merge when ready.

**Pull request merged**